### ISSUE TITLE
Component's children should not be mutated.

### ISSUE THREAD

**Issue by chintan9** [no23reason/react-geolocated#4]
```
05:57:12.132 Warning: Component's children should not be mutated.
    in td (created by Demo)
    in tr (created by Demo)
    in tbody (created by Demo)
    in table (created by Demo)
    in Demo (created by Geolocated(Demo))
    in Geolocated(Demo) (created by App)
    in div (created by App)
    in App1warning.js?:44:9warningwebpack:///./~/react/~/fbjs/lib/warning.js?:44:9handleElementwebpack:///./~/react/lib/ReactChildrenMutationWarningDevtool.js?:42:11ReactDOMUnknownPropertyDevtool.onComponentHasMountedwebpack:///./~/react/lib/ReactChildrenMutationWarningDevtool.js?:53:5emitEvent/<webpack:///./~/react/lib/ReactDebugTool.js?:30:9emitEventwebpack:///./~/react/lib/ReactDebugTool.js?:27:3ReactDebugTool.onComponentHasMountedwebpack:///./~/react/lib/ReactDebugTool.js?:254:5ReactDOMComponent.Mixin.mountComponent/callbackwebpack:///./~/react/lib/ReactDOMComponent.js?:574:18.notifyAllwebpack:///./~/react/lib/CallbackQueue.js?:67:9ON_DOM_READY_QUEUEING.closewebpack:///./~/react/lib/ReactReconcileTransaction.js?:81:5Mixin.closeAllwebpack:///./~/react/lib/Transaction.js?:204:11Mixin.performwebpack:///./~/react/lib/Transaction.js?:151:11Mixin.performwebpack:///./~/react/lib/Transaction.js?:138:13.performwebpack:///./~/react/lib/ReactUpdates.js?:90:12flushBatchedUpdateswebpack:///./~/react/lib/ReactUpdates.js?:173:7Mixin.closeAllwebpack:///./~/react/lib/Transaction.js?:204:11Mixin.performwebpack:///./~/react/lib/Transaction.js?:151:11ReactDefaultBatchingStrategy.batchedUpdateswebpack:///./~/react/lib/ReactDefaultBatchingStrategy.js?:63:7enqueueUpdatewebpack:///./~/react/lib/ReactUpdates.js?:201:5enqueueUpdatewebpack:///./~/react/lib/ReactUpdateQueue.js?:25:3ReactUpdateQueue.enqueueSetStatewebpack:///./~/react/lib/ReactUpdateQueue.js?:210:5ReactComponent.prototype.setStatewebpack:///./~/react/lib/ReactComponent.js?:64:3onPositionSuccesswebpack:///./~/react-geolocated/dist-modules/components/geolocated.js?:89:21componentDidMountwebpack:///./~/react-geolocated/dist-modules/components/geolocated.js?:102:21invokeComponentDidMountWithTimerwebpack:///./~/react/lib/ReactCompositeComponent.js?:60:3.notifyAllwebpack:///./~/react/lib/CallbackQueue.js?:67:9ON_DOM_READY_QUEUEING.closewebpack:///./~/react/lib/ReactReconcileTransaction.js?:81:5Mixin.closeAllwebpack:///./~/react/lib/Transaction.js?:204:11Mixin.performwebpack:///./~/react/lib/Transaction.js?:151:11batchedMountComponentIntoNodewebpack:///./~/react/lib/ReactMount.js?:126:3Mixin.performwebpack:///./~/react/lib/Transaction.js?:138:13ReactDefaultBatchingStrategy.batchedUpdateswebpack:///./~/react/lib/ReactDefaultBatchingStrategy.js?:63:7batchedUpdateswebpack:///./~/react/lib/ReactUpdates.js?:98:3ReactMount._renderNewRootComponentwebpack:///./~/react/lib/ReactMount.js?:285:5ReactMount._renderSubtreeIntoContainerwebpack:///./~/react/lib/ReactMount.js?:371:21ReactMount.renderwebpack:///./~/react/lib/ReactMount.js?:392:12<anonymous>webpack:///./src/index.js?:19:1<anonymous>http://localhost:3000/static/js/bundle.js:1318:2__webpack_require__http://localhost:3000/static/js/bundle.js:556:12hotCreateRequire/fnhttp://localhost:3000/static/js/bundle.js:87:20<anonymous>webpack:///multi_main?:4:18<anonymous>http://localhost:3000/static/js/bundle.js:589:2__webpack_require__http://localhost:3000/static/js/bundle.js:556:12<anonymous>http://localhost:3000/static/js/bundle.js:579:18<anonymous>http://localhost:3000/static/js/bundle.js:1:11
```

**Comment by no23reason** [comment]
Hello, thanks for filing an issue. However to fix it, I will need some more information. Can you please tell me more about how and where this warning is being thrown?

**Comment by chintan9** [comment]
[https://github.com/chintan9/react-d3](https://github.com/chintan9/react-d3)


i got when I was running my home page

**Comment by no23reason** [comment]
Ok thanks, I will look into it :)

**Comment by no23reason** [comment]
@chintan9 Unfortunately, I cannot run your web page, there seem to be some files missing, most notably the whole `scripts` folder and the `geoLoc.js` file referenced from the root component. Could you please commit and push those files to your repository, so that I can run the application? Otherwise I am unable to reproduce and fix the bug...

**Comment by chintan9** [comment]
ok

**Comment by no23reason** [comment]
As it has been more than a month and I still do not have enough information to reproduce this bug, I'm closing this issue. There has been some [bug](https://github.com/facebook/react/issues/7424) in React itslef regarding this warning, that will be fixed in 15.4.0 so I hope it fixes this issue as well.
